Directory.Packages.props

```diff
@@ -1,7 +1,11 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
+    <AspNetVersion>8.0.0-rc.2.23480.2</AspNetVersion>
```

Comment:
These version numbers can now be `8.0.0` now, right?

Reply:
Yup yup!